specification/cognitiveservices/OpenAI.Inference/models/completions.create.cadl

```diff
@@ -126,40 +117,30 @@ How many generations to create server side, and display only the best. Will not
 stream intermediate progress if best_of > 1. Has maximum value of 128.
 """)
     best_of?: int32;
-}
+};
 
-@doc("""
```

Comment:
This documentation was duplicated from the field so that's great that it's gone. Thanks!